### PR TITLE
Epic Chatbar Update

### PR DESCRIPTION
**Pull request by ghost**
cb3

Made the chatbar look nice, font size is no longer unnecessarily huge and ugly, adds buttons for OOC and Me and makes them look more button-like

:cl:
add: The chatbar now has OOC and Me buttons
tweak: The chatbar font-size is smaller
/:cl:

### CODE REVIEW

**Comment by LetterN** [comment]
Emotes for high rp uses. Speedmerge now.

**Comment by newfren** [comment]
Background colour looks very green in that screenshot.

File: code/__DEFINES/colors.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // This is eventually for wjohn to add more color standardization stuff like I keep asking him >:(
 
 #define COLOR_INPUT_DISABLED "#F0F0F0"
-#define COLOR_INPUT_ENABLED "#D3B5B5"
+#define COLOR_INPUT_ENABLED "#B5D3BD"
```
Comment (CosmicScientist) [inline code/__DEFINES/colors.dm]: Is it possible for you to make this a preference? I'll kneejerk all over that I don't want to lose pinky grey but realistically I'd rather pick between and then whinge that I don't have the option for white text on a dark background.

**Comment by CosmicScientist** [comment]
Could you put in a screencap of the old and new font sizes side by side with some sample text?

File: code/__DEFINES/colors.dm (hunk 1 above)
Comment (uraniummeltdown) [reply code/__DEFINES/colors.dm]: It's possible, but I'm not sure if it's worth making 2 new prefs for chatbar font color and background color

**Comment by uraniummeltdown** [comment]
cb1
cb2



Here is a side by side comparison of the new and old chatbars minus the color change and buttons, excuse the Hotkey Toggle thing the other screenshot is from Paradise because I didn't want to recompile.

Please laugh emoji for the first one, party emoji for the second one

**Comment by arsserpentarium** [comment]
Please,add LOOC.

**Comment by ExcessiveUseOfCobblestone** [comment]
but pink is cute~

**Comment by Spaceinaba** [comment]
this pr is epic

**Comment by epochayur** [comment]
why change for the sake of change

**Comment by uraniummeltdown** [comment]
well the main reason I did this was to add the OOC and Me buttons, if people don't like the other changes please like this comment

**Comment by epochayur** [comment]
i like the buttons i'm just butthurt about the color

**Comment by Xhuis** [comment]
Yeah, I don't like the color at all. I don't mind the buttons.

**Comment by Armhulen** [comment]
epic

**Comment by WJohn** [comment]
I'm not a huge fan of the color change myself. It's gonna make a lot of people angry for no good reason, so keep it pink.

File: code/__DEFINES/colors.dm (hunk 1 above)
Comment (kevinz000) [reply code/__DEFINES/colors.dm]: yeah well the new one looks bad to some people

**Comment by CosmicScientist** [comment]
texts combined
Your examples side by side for anyone who wants to click and see.

**Comment by uraniummeltdown** [comment]
Alright I reverted the color change, it now looks like the bottom of what CosmicScientist posted

**Comment by Jalleo** [comment]
Heh rather than letting it be a preference for what color people want the background of the chat bar to be.
I personally liked the font size slightly larger but I don't mind it being smaller.

**Comment by optimumtact** [comment]
take a screenshot in game or get lost, this requirement is to ensure you've tested your pr to a bare minimum.

**Comment by uraniummeltdown** [comment]
the OP picture IS ingame you boob, I'll take an updated one when I have access to a computer

**Comment by optimumtact** [comment]
boob or boob not, there is no try

**Comment by uraniummeltdown** [comment]
updated OP

**Pull request merged**